homeassistant/components/microbees/cover.py

```diff
@@ -0,0 +1,116 @@
+"""Cover integration microBees."""
+from typing import Any
+
+from microBeesPy import Actuator
+
+from homeassistant.components.cover import (
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.event import async_call_later
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the microBees cover platform."""
+    coordinator: MicroBeesUpdateCoordinator = hass.data[DOMAIN][
+        entry.entry_id
+    ].coordinator
+
+    async_add_entities(
+        MBCover(
+            coordinator,
+            bee_id,
+            next(filter(lambda x: x.deviceID == 551, bee.actuators)).id,
+            next(filter(lambda x: x.deviceID == 552, bee.actuators)).id,
+        )
+        for bee_id, bee in coordinator.data.bees.items()
+        if bee.productID == 47
```

Comment:
The library should create human readable abstractions for the product ids etc so that the library user doesn't need to handle these magic numbers.

One way would be to create enums that name the numbers.